src/applications/personalization/dashboard/containers/MessagingWidget.jsx

```diff
@@ -91,7 +91,7 @@ class MessagingWidget extends React.Component {
         {content}
         <p>
           <a
-            href={`${mhvBaseUrl()}/mhv-portal-web/secure-messaging`}
+            href={mhvUrl('secure-messaging')}
```

Comment:
This is overkill is this doesn't happen often, but you could abstract into an `MHVLink` React component -

```jsx
function MHVLink({ href, ...props }) {
    return <a href={mhvUrl(href)} {...props}/>
}
```

Reply:
there's only 2 or so instances of this outside of the `cta-widget` helpers, so probably not necessary at this point